### PR TITLE
integration: add TestV3WatchMultipleStreams

### PR DESCRIPTION
**Pull request by gyuho**
Related https://github.com/coreos/etcd/issues/4216.

### CODE REVIEW

File: integration/v3_grpc_test.go [hunk 1]
```diff
@@ -641,6 +642,77 @@ func (evs eventsSortByKey) Len() int           { return len(evs) }
...
+		if errW != nil {
+			t.Fatalf("wAPI.Watch error: %v", errW)
+		}
+		if err := wStream.Send(&pb.WatchRequest{CreateRequest: &pb.WatchCreateRequest{Key: []byte("foo")}}); err != nil {
+			t.Fatalf("wStream.Send error: %v", err)
+		}
+		streams[i] = wStream
+	}
+
+	for i := range streams {
```
Comment (heyitsanthony) [inline integration/v3_grpc_test.go]: `for _, wStream := ...` ?

Comment (gyuho) [inline integration/v3_grpc_test.go]: Ok will fix.

File: integration/v3_grpc_test.go [hunk 2]
```diff
@@ -641,6 +642,77 @@ func (evs eventsSortByKey) Len() int           { return len(evs) }
...
+				t.Errorf("wresp.Events got = %+v, want = %+v", wresp.Events, wevents)
+			}
+		}(i)
+	}
+	wg.Wait()
+
+	// now Recv should block because there is no more events coming
+	for i := range streams {
+		wStream := streams[i]
+		rok, nr := WaitResponse(wStream, 1*time.Second)
```
Comment (heyitsanthony) [inline integration/v3_grpc_test.go]: move to goroutine above so waits are concurrent?

File: integration/v3_grpc_test.go [hunk 3]
```diff
@@ -569,9 +570,9 @@ func TestV3WatchMultipleEventsFromCurrentRevision(t *testing.T) {
 	clus := newClusterGRPC(t, &clusterConfig{size: 3})
 
 	wAPI := pb.NewWatchClient(clus.RandConn())
-	wStream, err := wAPI.Watch(context.TODO())
-	if err != nil {
-		t.Fatalf("wAPI.Watch error: %v", err)
+	wStream, wErr := wAPI.Watch(context.TODO())
+	if wErr != nil {
+		t.Fatalf("wAPI.Watch error: %v", wErr)
```
Comment (gyuho) [inline integration/v3_grpc_test.go]: fixing this based on `go tool -shadow`.

**Comment by gyuho** [comment]
@heyitsanthony Just addressed. PTAL. Thanks!

**Comment by heyitsanthony** [comment]
lgtm

**Comment by gyuho** [comment]
Thanks

**Pull request merged**